shell/platform/android/io/flutter/plugin/platform/PlatformViewsController.java

```diff
@@ -144,6 +144,12 @@ public void createAndroidViewForPlatformView(
           }
 
           final PlatformView platformView = factory.create(context, request.viewId, createParams);
+          // If our FlutterEngine is already attached to a Flutter UI, provide that Android
+          // View to this new platform view.
+          if (flutterView != null) {
+            platformView.onFlutterViewAttached(flutterView);
```

Comment:
The platform view isn't attached at this point. It might need to go in `initializePlatformViewIfNeeded`.